src/main/cms/cms_menu_imu.c

```diff
@@ -285,7 +285,7 @@ static long cmsx_profileOtherOnExit(const OSD_Entry *self)
 static OSD_Entry cmsx_menuProfileOtherEntries[] = {
     { "-- OTHER PP --", OME_Label, NULL, pidProfileIndexString, 0 },
 
-    { "D SETPT WT",  OME_FLOAT,  NULL, &(OSD_FLOAT_t)  { &cmsx_dtermSetpointWeight,    0,    255,   1, 10 }, 0 },
+	{ "D SETPT WT",  OME_UINT16, NULL, &(OSD_UINT16_t) { &cmsx_dtermSetpointWeight,    0,    2000,  1, 10 }, 0 },
```

Comment:
The `10` in this line is extra and causes the build to fail. It is used for `OME_FLOAT` to indicate the divisor, but not used for `OME_UINT16`.

Reply:
Yes, already repaired :)